src/api/user/README.md

```diff
@@ -0,0 +1,90 @@
+# User Service
+
+The User Service provides a means of communication between backend and Firestore via http requests. It is intended to receive http requests containing user data (_*validated for consistency*_) which is then persisted to Google's Firestore database.
+
+## Info
+
+There are several moving pieces to this micro-service which may deceptively seem confusing upon first glance. The steps from request to persistence are as follows:
+
+1. Express is the 'conductor' of the microservice. It receives http requests from users (_i.e._ the frontend) containing data regarding a Telescope user (_see:_ `src/models/user.js`).
+2. This data is validated by [celebrate](https://www.npmjs.com/package/celebrate) (_see:_ `src/models/celebrateSchema.js`) to ensure consistent integrity.
+3. Only when all data is accepted is a user object constructed (_see:_ `src/models/user.js`) and persisted to our db (_see:_ `src/routes/user.js`).
+
+Since Firestore requires unique private API keys in order to work with the production (online) version of the database, the Firestore emulator has instead been localized and is ready for dev use. This ensures that code that passes tests locally will function as intended remotely. If (_when_) changes are required, simply create your change (and accompanying unit test), and run the test runner.
+
+Our services contain a dockerized version of the Firestore Emulator for use, but an undockerized version can also be ran (see Usage).
+
+### About Firestore
+
+Firestore is a NoSQL database for mobile, web, and server development from Firebase and Google Cloud. Firestore is comprised of collections (which hold documents) and documents (which hold NoSQL data).
+
+More info can be found on the [Firebase documentation page](https://firebase.google.com/docs/firestore), or for the more visually inclined [on the youtube page](https://www.youtube.com/playlist?list=PLl-K7zZEsYLluG5MCVEzXAQ7ACZBCuZgZ).
+
+## Requirements
+
+\- Node.js - version 8.0 or higher.
+
+\- Java - version 1.8 or higher (used with the Emulator.)
+
+The Local Emulator Suite must be installed on your machine in order to run the Firestore emulator. Information on how to install it can be found [here](https://firebase.google.com/docs/emulator-suite/install_and_configure) (or via `npm install -g firebase-tools`).
+
+## Install
+
+from inside /api/src/user:
+
+```
+npm install
+```
+
+from root:
+
+```
+npm install:user-service
+```
+
+## Usage
+
+```
+# normal mode (to be fixed with final prod structure)
+npm start
+
+# running firestore emulator locally
```

Comment:
I don't think people will do this.  We'll use the dockerized version, so you'd do:

```
npm run services:start firebase
npm test (or start)
```